packages/bsb-core/tests/test_postprocessing.py

```diff
@@ -255,7 +256,7 @@ def test_multiple_roots(self):
         in C > A and B > A"""
         self.cfg.after_connectivity = dict(
             new_connection=dict(
-                strategy="merge_connections",
+                strategy=bsb.postprocessing.MergeDirect,
                 connections=["D_to_A", "C_to_D", "B_to_D"],
             )
```

Comment:
just fyi that that is equivalent to simply `bsb.postprocessing.MergeDirect(connections=["D_to_A", "C_to_D", "B_to_D"])`